src/physics/mod.rs

```diff
@@ -23,7 +159,6 @@ pub fn insert_object_colliders(
     object_entity: Entity,
```

Comment:
I believe this function should be removed and replaced by the `PhysicsBackend` functions call, as you did for the `insert_tile_colliders`.
However, you should have a look at `src/properties/events.rs`, I think you missed some functions to replace there (and this is why CI is not happy).

For this special case, instead of relying upon the `TiledMapSettings` component, you could directly pass the `PhysicsBackend` enum to both `spawn_collider` functions (ie. as it is done for `ColliderCallback`).